EpicLoot/ItemDrop_Patch_MagicItemTooltip.cs

```diff
@@ -655,7 +669,14 @@ public static class AugaTooltipPreprocessor
                     }
                 }
             }
-            
+
+            switch (label)
```

Comment:
Why switch on one label?

Reply:
Switch on one label is for future growth to be honest, and to follow the pattern, but I needed it outside of the IsMagic if statement.

Reply:
Sounds good to me